rotkehlchen/db/drivers/gevent.py

```diff
@@ -412,3 +427,75 @@ def total_changes(self) -> int:
         """total number of database rows that have been modified, inserted,
         or deleted since the database connection was opened"""
         return self._conn.total_changes
+
+    def schema_sanity_check(self) -> None:
+        """Ensures that database schema is not broken. Raises DBSchemaError if anything is bad."""
+        # Get mapping that contains expected database structure
+        expected_minimized_schema = MINIMIZED_SCHEMA_MAP.get(self.connection_type)
```

Comment:
you don't need this. The schema is a propery of the Connection. Initiaize it at connection init. Don't do it each time thesanity check happens.